torch/distributed/rpc/internal.py

```diff
@@ -119,6 +119,9 @@ def _run_function(binary_data, tensor_table):
         result = RemoteException(except_str)
     return result
 
+def _handle_exception(result):
+    if isinstance(result, RemoteException):
```

Comment:
question/nit, do we lost the original Exception type while wrapping it as `RemoteException`? If we are able to retrieve the original type, rethrowing it as the original type will be helpful.

Reply:
@zzzwen all exception type, msg and stack trace are wrapped into remoteException and users can read these info in client.